examples/pubsub/README.md

```diff
@@ -19,12 +19,19 @@ docker run -d --rm --hostname my-rabbitmq --name my-rabbitmq \
     rabbitmq:3-management
 
 # Run the example
+dapr run --app-protocol http
```

Comment:
@hhunter-ms, I guess we should remove this first command as it only loads the side car (without application) and we have the actual commands below.

Reply:
@shubham1172 Ope, thanks for catching that! I forgot to remove